docs/usage.md

```diff
@@ -475,6 +480,101 @@ We recommend adding the following line to your environment to limit this (typica
 NXF_OPTS='-Xms1g -Xmx4g'
 ```
 
+## Tutorials
+
+### Tutorial - How to create your custom database
+
+#### Kraken2
+
+Kraken2 allows the user to build custom databases.
+
+To install a taxonomy:
+
+```bash
+kraken2-build --download-taxonomy --db $DBNAME
+```
+
+To install one or more reference libraries:
+
+```bash
+--download-library bacteria --db $DBNAME
+--download-library viral --db $DBNAME
+--download-library archaea --db $DBNAME
+```
+
+To add more genomes:
+
+```bash
+kraken2-build --add-to-library genome.fa --db $DBNAME
+```
+
+You can follow Kraken2 [tutorial](https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#custom-databases) for a more detailed description.
```

Comment:
```suggestion

> These are instructions are based on Kraken 2.1.2

To build a Kraken2 database you need two components: a taxonomy (consisting of `names.dmp`, `nodes.dmp`, and `*accession2taxid`) files, and the FASTA files you wish to include.

To install pull the NCBI taxonomy you can run the following:

```bash
kraken2-build --download-taxonomy --db <YOUR_DB_NAME>
```

You can then add your FASTA files with the following build command

```bash
kraken2-build --add-to-library *.fna --db <YOUR_DB_NAME>
```

> 🛈 You can repeat this step multiple times to iteratively add more genomes prior building

You can also automatical download and add 'standard' libraries provided by Kraken2 (e.g. bacteria on RefSeq)

```bash
kraken2-build --download-library bacteria --db <YOUR_DB_NAME>
```

Once all genomes are added to the library, you can build the database (and optionally clean it up):

```bash
kraken2-build --build --db <YOUR_DB_NAME>
kraken2-build --clean--db <YOUR_DB_NAME>
```

You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database input sheet.

You can follow the Kraken2 [tutorial](https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#custom-databases) for a more detailed description.
```